en/docs/observe-and-manage/setting-up-cloud-native-observability-in-kubernetes.md

```diff
@@ -194,14 +194,24 @@ flow.statistics.capture_all= true
 stat.tracer.collect_payloads= true
 stat.tracer.collect_mediation_properties= true
 
-[opentracing]
+[opentelemetry]
 enable = true
-jaeger.sampler.manager_host="hostname"
-jaeger.sender.agent_host="hostname"
+logs = true
+type = "jaeger"
+host = "<hostname-of-jaeger-endpoint>"
+port =  "<port-of-jaeger-endpoint>"
 ```
 
-!!! tip
-    Enter the host name of your Jaeger service as the value for `manager_host` and `agent_host` parameters. 
+!!! Tip
+    Instead of using `host` and `port`, the `url` can be used directly to connect to Jaeger in the following way.
+
+    ```toml
+    [opentelemetry]
+    enable = true
+    logs = true
+    type = "jaeger"
+    url =  "<url-of-jaeger-endpoint>"
+    ```
```

Comment:
```suggestion
> 💡 **Tip**: Instead of using `host` and `port`, the `url` can be used directly to connect to Jaeger in the following way:
>
> ```toml
> [opentelemetry]
> enable = true
> logs = true
> type = "jaeger"
> url =  "<url-of-jaeger-endpoint>"
> ```
```

Reply:
Your preview looks 
![Screenshot 2024-09-11 at 10 53 52](https://github.com/user-attachments/assets/54d567ef-3398-4367-aad7-9d5819a85146)

Reply:
Hi @pcnfernando, since this is a MkDocs website, we must format the notes according to the framework.

For more info see the [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/reference/admonitions) documentation.